syntax = "proto3";
package m2g.pb;

enum ObjectType {
  NO_OBJECT = 0;

  HUMAN_PLAYER = 1;
  INCOME_LEVEL_MARKER = 2;
}
